retry.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Retry a command a few times if it fails
+# https://github.com/konflux-ci/build-trusted-artifacts/tree/main/retry.sh
+
```

Comment:
One suggestion to acknowledge @arewm 's proposal, add a comment here.

For example:

```suggestion
    # Note: There is a similar retry script in https://github.com/konflux-ci/oras-container/blob/main/hack/retry.sh
    # Potentially we could be use that shared base image here and consolidating
    # way the retries are done
```

Reply:
Applied, thank you!